### ISSUE TITLE
Installation is Vague

### ISSUE THREAD

**Issue by damassi** [seanho/SimpleView#5]
Could you please detail the steps to install the gem in your gemfile?  gem install simple-view does not work, and 'your gemfile' is somewhat vague.

**Comment by damassi** [comment]
Any update on this?  Following the instructions (even after installing bundler, etc.) simple-view is still not found.  

In order for my project to compile, I had to grab your includes from the root (while adjusting paths):

app.files += Dir.glob(File.join(app.project_dir, 'vendor/SimpleView/_.rb'))
  app.files.unshift(Dir.glob(File.join(app.project_dir, 'vendor/SimpleView/builders/_.rb')))
  app.files.unshift(File.join(app.project_dir, 'vendor/SimpleView/builders/ui_control_builder.rb'))
  app.files.unshift(File.join(app.project_dir, 'vendor/SimpleView/builders/ui_view_builder.rb'))
  app.files.unshift(Dir.glob(File.join(app.project_dir, 'vendor/SimpleView/builders/helpers/_.rb')))
  app.files.unshift(Dir.glob(File.join(app.project_dir, 'vendor/SimpleView/extensions/_.rb')))

Curious as to why you're going this route rather than the (seemingly) more simple and common gem install package route?

**Comment by siuying** [comment]
1. You need to install bundler
2. Create `Gemfile` with following content:

``` ruby
source :rubygems
gem 'simple-view', :git => 'https://github.com/seanho/SimpleView.git'
```
1. Install the bundle: `bundle install`
2. Load bundler in your app:

``` ruby
$:.unshift("/Library/RubyMotion/lib")
require 'motion/project'
require 'rubygems'
require 'bundler'
Bundler.require :default

Motion::Project::App.setup do |app|
  # ...
end
```

**Comment by damassi** [comment]
Appreciated; these instructions worked.  This seems like it should be documented on the README, however, particularly for new (new) users who are unfamiliar with Bundler and how it interfaces with the RM build-process.

**Comment by siuying** [comment]
I believe every steps above are actually presented in readme, perhaps what you missing is bundle install?

Yep I agreed this could be difficult for ppl not familiar on bundler, perhaps a gem release will help.

**Comment by damassi** [comment]
I agree.  It's good to remember that due to RubyMotion's accessibility, a lot of people are going to be coming to it without a formal Ruby background.  An installation similar to BubbleWrap would be ideal.
